torchvision/utils.py

```diff
@@ -382,6 +389,130 @@ def draw_keypoints(
     return torch.from_numpy(np.array(img_to_draw)).permute(2, 0, 1).to(dtype=torch.uint8)
 
 
+# Flow visualization code adapted from https://github.com/tomrunia/OpticalFlow_Visualization
+@torch.no_grad()
+def flow_to_image(
```

Comment:
Maybe we should name this as `draw_optical_flow` ?

Reply:
`flow_to_image` sounds good to me: it's in line with our current utilities like "pil_to_tensor()", etc.